data/lua/core/interface.lua

```diff
@@ -5,8 +5,11 @@ if wesnoth.kernel_type() == "Game Lua Kernel" then
 
 	wesnoth.interface.select_unit = wesnoth.units.select
 
-	--! Fakes the move of a unit satisfying the given @a filter to position @a x, @a y.
-	--! @note Usable only during WML actions.
+	-- Fakes the move of a unit satisfying the given filter to position x, y.
+	-- Usable only during WML actions.
```

Comment:
Should these lines start with a triple-hyphen?

Reply:
I think the triple hyphen is required for the `@` directives but not for general documentation… still, I should probably keep it consistent.